modules/ROOT/pages/quickstart.adoc

```diff
@@ -93,14 +93,14 @@ For example, if you see a package that is listed in the `-gate` tag but not the
 
 === Pungi and the Compose Configs
 
-If you want to make changes to the overall distribution itself, or the artifacts produced by composes, or you’ll want to refer to the `Distribution` component in Bugzilla. Please file an appropriate bugzilla describing the change you’d like to make.
+If you want to make changes to the overall distribution itself, or the artifacts produced by composes, or you’ll want to refer to the `Distribution` component in the RHEL project. Please file an appropriate issue describing the change you’d like to make.
```

Comment:
s/or you'll want/you'll want/

Not introduced in this change but let's fix the sentence while at it.